epam-assets/scss/control-mods.scss

```diff
@@ -1,6 +1,6 @@
 @use './effects' as *;
-@use './loveship-color-vars' as *;
 @use './control-layout' as *;
```

Comment:
check could we delete this file, because it seams that isn't used?

Reply:
it's not used - deleted